packages/odata-common/src/edm-types.ts

```diff
@@ -1,10 +1,11 @@
 import { ODataVersion } from '@sap-cloud-sdk/util';
 /**
- * @internal
+ * Contains all edm types which are shared between different odata versions.
  */
 export type EdmTypeCommon = EdmTypeSameConverters | EdmTypeDifferentConverters;
+
 /**
- * @internal
+ * PLEASER REVIEW.
```

Comment:
[q] Why does this have to be exported?
```suggestion
 * Includes all EDM types that have the same converters for OData version 2 and 4.
```

Reply:
Same as EdmTypeDifferentConverters